pages/search/search.scss

```diff
@@ -16,4 +16,40 @@
     display: flex;
     align-items: center;
   }
+
+  .help-dropdown {
+    @include form-control( $black );
```

Comment:
Not to beat a dead horse, but this is the styling that actually gets built here- it looks like this mixin is effectively only applying a border-radius and border-color at larger sizes. Something just doesn't feel right.
<img width="293" alt="image" src="https://user-images.githubusercontent.com/1682681/60998292-9d9e6900-a326-11e9-8ac5-8a16dd46f614.png">

Reply:
Oh, I see where the extra styles come in. It's a little less weird now that I realize that.

Reply:
yeah, you're right, the mixin doesn't play a huge role here and can be removed.